integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/PhraseSearchPredicateIT.java

```diff
@@ -194,6 +198,92 @@ public void noTokenAfterAnalysis() {
 				.hasNoHits();
 	}
 
+	@Test
+	public void analyzerOverride() {
+		StubMappingSearchScope scope = indexManager.createSearchScope();
+
+		String whitespaceAnalyzedField = indexMapping.whitespaceAnalyzedField.relativeFieldName;
+		String whitespaceLowercaseAnalyzedField = indexMapping.whitespaceLowercaseAnalyzedField.relativeFieldName;
+
+		IndexSearchQuery<DocumentReference> query = scope.query()
+				.asReference()
+				.predicate( f -> f.phrase().onField( whitespaceAnalyzedField ).matching( "ONCE UPON" ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_2 );
+
+		query = scope.query()
+				.asReference()
+				.predicate( f -> f.phrase().onField( whitespaceLowercaseAnalyzedField ).matching( "ONCE UPON" ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_1, DOCUMENT_2, DOCUMENT_3 );
+
+		query = scope.query()
+				.asReference()
+				.predicate( f -> f.phrase().onField( whitespaceAnalyzedField ).matching( "ONCE UPON" )
+						.analyzer( OverrideAnalysisDefinitions.ANALYZER_WHITESPACE_LOWERCASE.name ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_1 );
+	}
+
+	@Test
+	public void analyzerOverride_notExistingName() {
+		StubMappingSearchScope scope = indexManager.createSearchScope();
+		String whitespaceAnalyzedField = indexMapping.whitespaceAnalyzedField.relativeFieldName;
+
+		SubTest.expectException( () -> scope.query()
+				.asReference()
+				.predicate( f -> f.phrase().onField( whitespaceAnalyzedField ).matching( "ONCE UPON" )
+						// we don't have any analyzer with that name
+						.analyzer( "this_name_does_actually_not_exist" ) )
+				.toQuery().fetch()
+		)
+				.assertThrown()
+				.isInstanceOf( SearchException.class )
+
+		// TODO on Elasticsearch backend the error is not handled by Search
+		//.hasMessageContaining( "HSEARCH000353: Unknown analyzer: 'this_name_does_actually_not_exist'. Make sure you defined this analyzer." )
+		;
+	}
+
+	@Test
+	public void analyzerIgnore() {
```

Comment:
Same comment about skipping analysis for normalized fields: currently the option is silently ignored for normalized fields, that's probably not a good idea.

In this specific case I would just remove the option for phrase predicates, because it isn't very useful and we don't want to waste time.